wrappers/core18_test.go

```diff
@@ -48,6 +48,8 @@ func makeMockSnapdSnap(c *C) *snap.Info {
 	c.Assert(err, IsNil)
 	err = os.MkdirAll(dirs.SnapDBusSessionPolicyDir, 0755)
 	c.Assert(err, IsNil)
+	err = os.MkdirAll(dirs.SnapDesktopFilesDir, 0755)
+	c.Assert(err, IsNil)
```

Comment:
as @jhenstridge as remarked the code should maybe do this by itself

Reply:
All the other directories are made in this test - should this be a special case?

Reply:
I think the tests should not create directories that the code also has code to create otherwise,  how is that code tested or we are sure it will not be removed later? another approach is to have test variants with the directory already present or not but given how MkdirAll works that might be overkill